gensim/parsing/porter.py

```diff
@@ -44,14 +67,46 @@ def __init__(self):
 
         Note that only lower case sequences are stemmed. Forcing to lower case
         should be done before stem(...) is called.
-        """
 
+        Examples
+        --------
+        >>> from gensim.parsing.porter import PorterStemmer
+        >>> p = PorterStemmer()
+        >>> print "b (word) = ", p.b, " ,k (readjusted downwards as the stemming progresses) = ", p.k, " ,j (word length) = ", p.j
+        b =    ,k =  0  ,j =  0
+
+        """
         self.b = ""  # buffer for word to be stemmed
         self.k = 0
         self.j = 0   # j is a general offset into the string
 
     def _cons(self, i):
-        """True <=> b[i] is a consonant."""
+        """Take b[i], check if it is a consonant.
```

Comment:
In class description we already have:
b : str : is a buffer holding a word to be stemmed. The letters are in b[0], b[1] ... ending at b[k].

Probably it will be enough to add "letter" after "... it is a consonant".